sol-transactions/templates/README.md.gotmpl

```diff
@@ -0,0 +1,46 @@
+# Solana Transactions
+
+This Substreams project allows you to retrieve Solana transactions filtered by one or several Program IDs (i.e. you will only receive transactions containing the specified Program IDs).
+**NOTE:** Transactions containing voting instructions will NOT be present.
+
+## Understand the Generated Project
+
+Only a `substreams.yaml` file has been generated. This file declare a Substreams module, `map_filtered_transactions`, which uses the Solana Foundational Modules (module built by the team).
+
+```yaml
+specVersion: v0.1.0
+package:
+  name: my_project_sol
+  version: v0.1.0
+
+imports:
+  solana: https://spkg.io/streamingfast/solana-common-v0.2.0.spkg // 1.
+
+modules:
+ - name: map_filtered_transactions // 2.
+   use: solana:filtered_transactions_without_votes // 3.
+
+network: solana
+
+params:
+  map_filtered_transactions: {{ .ProgramId }} // 4.
+```
+1. Import the Solana Foundational Modules.
+2. Declare the `map_filtered_transactions` module, which you will run later.
+3. Use the `filtered_transactions_without_votes` module from the Solana Foundational Modules.
+Essentially, you are _using_ the Solana Foundational Module, which is pre-built for you.
+4. Pass the regular expression to filter the transactions based on the specified Program IDs.
+
+## Authenticate
```

Comment:
These "next steps" commands should be first up there, with only the commands like:
```bash
substreams build
substreams auth
substreams gui
```

as a continuation of your onboarding.

And if this module leads an easy way to subgraphs, thne we can add: `substreams codegen subgraph`